src/elements/content-sidebar/AddTaskButton.js

```diff
@@ -51,13 +53,15 @@ class AddTaskButton extends React.Component<Props, State> {
 
     handleClickMenuItem = (taskType: TaskType) => this.setState({ isTaskFormOpen: true, taskType });
 
-    handleModalClose = () => this.setState({ isTaskFormOpen: false });
+    handleModalClose = () => this.setState({ isTaskFormOpen: false, error: null });
 
-    handleSubmit = () => this.setState({ isTaskFormOpen: false });
+    handleCreateSuccess = () => this.setState({ isTaskFormOpen: false });
```

Comment:
Do we also need to clear the error state here if the first task fails but the second one succeeds?

Reply:
yeah good catch